### PR TITLE
Added new article by Mark Erikson

### PR DESCRIPTION
**Pull request by TheLarkInn**
Declaratively Rendering Earth in 3D, Part 1: Building a Cesium + React App with Webpack

### CODE REVIEW

**Comment by joshwiens** [comment]
Joo must sign the CLA of doom :)

**Comment by TheLarkInn** [comment]
Uh wat. Lol

**Comment by TheLarkInn** [comment]
Done haha forgot I hadn't signed here

File: README.md [hunk 1]
```diff
@@ -213,6 +213,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 - Jonathan Creamer | 02-Sept-15 - [Advanced WebPack Part 1 - The CommonsChunk Plugin.](http://jonathancreamer.com/advanced-webpack-part-1-the-commonschunk-plugin/)
 - Maxime Fabre | 16-Oct-15 - [Webpack your bags.](https://blog.madewithlove.be/post/webpack-your-bags/?utm_content=buffer480f4&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer)
 - Andrew Ray | 09-Apr-16 - [Webpack: When To Use and Why.](http://blog.andrewray.me/webpack-when-to-use-and-why/)
+- Mark Erikson | 07-Mar-17 - [Declaratively Rendering Earth in 3D, Part 1: Building a Cesium + React App with Webpack](http://blog.isquaredsoftware.com/2017/03/declarative-earth-part-1-cesium-webpack/)
```
Comment (d3viant0ne) [inline README.md]: Articles / Videos are both listed by publication date from newest to oldest.

**Pull request merged**